### PR TITLE
Suggestions

### PR DESCRIPTION
**Pull request by bobpace**

### CODE REVIEW

File: nodeCf.js [hunk 1]
```diff
@@ -1,176 +1,207 @@
-var Promise = require('bluebird');
-var nunjucks = require("nunjucks");
-var _ = require('lodash');
-var Ajv = require('ajv');
-var fs = require('fs');
+const Promise = require('bluebird');
```
Comment (CoreyKaylor) [inline nodeCf.js]: Wouldn't this be even better as an import?

Comment (bobpace) [inline nodeCf.js]: Doesn't look like these are supported out of the box with node v7.9.0 yet
https://medium.com/the-node-js-collection/an-update-on-es6-modules-in-node-js-42c958b890c

Comment (jaymell) [inline nodeCf.js]: I'm certainly not opposed to using a transpiler to get that functionality, but thus far haven't given it much thought.

**Comment by jaymell** [comment]
Thanks. This is beautiful. Can you look over and merge [this branch](https://github.com/jaymell/nodeCf/tree/suggestionsSuggestion) to yours first before I merge this? It fixes a couple minor issues. Everything deploys fine with those changes.

**Pull request merged**